### PR TITLE
ocamlPackages.mirage-random: 2.0.0 -> 3.0.0

### PR DESCRIPTION
**Pull request by StridBot**
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://opam.ocaml.org/packages/mirage-random/mirage-random.3.0.0.

meta.description for ocamlPackages.mirage-random is: "Random signatures for MirageOS"

meta.homepage for ocamlPackages.mirage-random is: "https://github.com/mirage/mirage-random"

meta.changelog for ocamlPackages.mirage-random is: ""

###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/mirage/mirage-random/releases/tag/v3.0.0)

- [Compare changes on GitHub](https://github.com/mirage/mirage-random/compare/v2.0.0...v3.0.0)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- Warning: a test defined in `passthru.tests` did not pass

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 3.0.0 with grep in /nix/store/qyaq17gfi8z5l5g7nfknxnrfwkm5r2pb-ocaml4.13.1-mirage-random-3.0.0
- found 3.0.0 in filename of file in /nix/store/qyaq17gfi8z5l5g7nfknxnrfwkm5r2pb-ocaml4.13.1-mirage-random-3.0.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

0 x86_64-linux rebuild(s)
0 x86_64-darwin rebuild(s)


First fifty rebuilds by attrpath
ocamlPackages.mirage-random
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Build yourself:
```
nix-build -A ocamlPackages.mirage-random https://github.com/StridBot/nixpkgs/archive/04c78695b9a4fbfe355d318a18050347f14a6ae8.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/qyaq17gfi8z5l5g7nfknxnrfwkm5r2pb-ocaml4.13.1-mirage-random-3.0.0
ls -la /nix/store/qyaq17gfi8z5l5g7nfknxnrfwkm5r2pb-ocaml4.13.1-mirage-random-3.0.0/bin
```

---

</details>
<br/>

### Pre-merge build results

NixPkgs review skipped

---

###### Maintainer pings

cc @vbgl for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
@ulrikstrid also CI did not trigger here

**Comment by ulrikstrid** [comment]
@SuperSandro2000 I amended and pushed so now it's triggered (sorry for the ping spam)

File: pkgs/development/ocaml-modules/mirage-random/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 buildDunePackage rec {
   pname = "mirage-random";
-  version = "2.0.0";
+  version = "3.0.0";
 
   useDune2 = true;
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/mirage-random/default.nix:7]: This line can be dropped.

Comment (ulrikstrid) [reply pkgs/development/ocaml-modules/mirage-random/default.nix]: Done

**Pull request merged**